scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -420,6 +437,35 @@ ALTER ROLE web_anon SET search_path TO grest, public;
 COMMIT;
 SQL
 
+
+# Data Types are intentionally kept varchar for single ID row to avoid future edge cases
+
+echo -e "\e[32mAdding initial genesis table..\e[0m"
+psql cexplorer >/dev/null << 'SQL'
+BEGIN;
+
+DROP TABLE IF EXISTS grest.genesis;
+CREATE TABLE grest.genesis (
+  NETWORKMAGIC varchar,
+  NETWORKID varchar,
+  ACTIVESLOTCOEFF varchar,
+  UPDATEQUORUM varchar,
+  MAXLOVELACESUPPLY varchar,
+  EPOCHLENGTH varchar,
+  SYSTEMSTART varchar,
+  SLOTSPERKESPERIOD varchar,
+  SLOTLENGTH varchar,
+  MAXKESREVOLUTIONS varchar,
+  SECURITYPARAM varchar,
+  ALONZOGENESIS varchar
+);
+
+COMMIT;
+SQL
+
+psql cexplorer -c "INSERT INTO grest.genesis VALUES ( '${SHGENESIS[4]}', '${SHGENESIS[2]}', '${SHGENESIS[0]}', '${SHGENESIS[1]}', '${SHGENESIS[3]}', '${SHGENESIS[5]}', '${SHGENESIS[6]}', '${SHGENESIS[7]}', '${SHGENESIS[8]}', '${SHGENESIS[9]}', '${SHGENESIS[10]}', '${ALGENESIS}' );"
```

Comment:
Split it into multiple lines for readability?

Reply:
> This will fail unless Alonzo genesis hash is specified in `config.json` and the file exists at the location specified - are we okay with that now

The node itself will fail as env on alpha branch has a check to ensure node version is > 1.29.
I am not too sure about readability on script, as then we have a lot of lines to edit/wrap down. But more importantly, wrapping there starts interfering with indentation /collapse tools - unless done consistently across the board